nbs/introduction.ipynb

```diff
@@ -85,152 +85,153 @@
    "execution_count": null,
```

Comment:
<p>I would rewrite the second sentense "A query is essentially a way to retrieve specific information from a dataset, i.e it finds all instantiations of the free variables that satisfy the query. " the second part of the sentence is not an example of the first part, so the use of "ie" is incorrect.</p><p> </p><p>Maybe just say that querying in rgxlog uses the same synatx and semantics as DataLog. Under said semantics, we try to find all instantiations of free variables that satisfy the queried relation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spanner_workbench/pull/134/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>